### PR TITLE
Gh 925/parse rejection transaction

### PR DESCRIPTION
**Pull request by MouazAlzahabi**

### CODE REVIEW

File: Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift [hunk 1]
```diff
@@ -281,6 +281,24 @@ class TransactionDetailCellBuilder {
                 detail: "\(customTx.dataSize.value) bytes")
             buildActions(tx)
             buildHexData(tx)
+        case .rejection(let rejectionTx):
+            // TODO: This should be changed when implemet display rejection transaction
```
Comment (sche) [inline Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift]: How about #warning macro?

File: Multisig/UI/Transaction/TransactionListViewController/TransactionListViewController.swift [hunk 2]
```diff
@@ -325,6 +325,19 @@ class TransactionListViewController: LoadableViewController, UITableViewDelegate
                 image = #imageLiteral(resourceName: "ico-custom-tx")
             }
             info = customInfo.methodName ?? ""
+        case .rejection(let rejectionInfo):
+            // TODO: This should be changed when implemet display rejection transaction
```
Comment (sche) [inline Multisig/UI/Transaction/TransactionListViewController/TransactionListViewController.swift]: Could you move code duplicate in a separate method?

**Review by DmitryBespalov — APPROVED** [review]

**Comment by sche** [comment]
Test

**Review by sche — CHANGES_REQUESTED** [review]

File: Multisig/UI/Transaction/TransactionListViewController/TransactionListViewController.swift (hunk 2 above)
Comment (MouazAlzahabi) [reply Multisig/UI/Transaction/TransactionListViewController/TransactionListViewController.swift]: This is temp code and will be replaced in the next ticket pr, that's why I kept it like this for now

File: Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift]: Done

**Pull request merged**